pyarn/lexer.py

```diff
@@ -15,36 +15,36 @@
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 """
 tokens = (
-    'STRING',
```

Comment:
Why `"` are wrong?

Reply:
Black formatting changes maybe? If so, I wonder if we could do all of that in a separate commit

Reply:
I think we should ask flake8 :sweat_smile:

Reply:
ah, it's black